### DISCUSSION TITLE
Saleor 3.0 Dashborard Staff Members access by Channel

### DISCUSSION THREAD

**Discussion by brodguez** [saleor/saleor#7773] [Q&A]
Is there a way on Saleor 3.0, on the dashboard, to give Staff Members access by Channels. In my case I have 2 channels, one for Store 1 and another for Store 2. Those 2 stores are not related I would like to have separated access for staff members. This way Staff from Store 1 will not see any data from Store 2 on the dashboard.
In case this is not implemented right now, is there a plan to do it?

**Comment by karolkielecki** [comment]
@brodguez Hi there! 

Thanks for this question. Assigning staff members to specific channels is not available. We'll consider how to solve such a case after delivering Saleor 3.0. Also, you'll be able to filter your products and orders by channel--I think it will be available in beta soon. 

Cheers,
